spec/defines/vhost_spec.rb

```diff
@@ -383,6 +383,7 @@
           'passenger_app_env'           => 'test',
           'passenger_ruby'              => '/usr/bin/ruby1.9.1',
           'passenger_min_instances'     => '1',
+          'passenger_max_requests'      => '1000',
```

Comment:
[Upstream](https://www.phusionpassenger.com/library/config/apache/reference/#passengermaxrequests) defaults to 0, not 1000.